portalnet/src/overlay/service.rs

```diff
@@ -1029,7 +1030,7 @@ where
         let content_keys: Vec<TContentKey> = request
             .content_keys
             .into_iter()
-            .map(|k| (TContentKey::try_from)(k))
+            .map(|k| (TContentKey::try_from)(k.as_ssz_bytes().into()))
```

Comment:
nit: Can't this be: `TContentKey::try_from(k)`?

In which case, entire line might be:

```rust
.map(TContentKey::try_from)
```